crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -444,6 +444,8 @@ en:
       identify: 'Identify'
     form:
       allocate: 'Allocate'
+      confirm_allocate: 'Allocating the node will wipe all the disks attached to the node including the shared disks. Proceed?'
```

Comment:
"Allocating the node will format all disks attached to this node, including shared disks. Do you want to proceed and allocate this node?"